### PR TITLE
Remove deprecated nimble service

### PR DESCRIPTION
**Pull request by wty-Bryant**

### CODE REVIEW

File: .changelog/4490529bdb8044068d99216cba424862.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+    "id": "4490529b-db80-4406-8d99-216cba424862",
+    "type": "feature",
+    "description": "Remove deprecated `nimble` service.",
+    "modules": [
+        "."
```
Comment (Madrigal) [inline .changelog/4490529bdb8044068d99216cba424862.json:6]: nit: shouldn't this be scoped to `nimble` instead of `.`?

**Review by Madrigal — APPROVED** [review]
If available, we should link any public announcement on these service deprecations

File: .changelog/4490529bdb8044068d99216cba424862.json (hunk 1 above)
Comment (lucix-aws) [reply .changelog/4490529bdb8044068d99216cba424862.json:6]: So technically the `nimble` module is being deleted -- I'm not sure if our tooling is written to handle that as an edge case. I suspect it will try to go update `service/nimble/CHANGELOG.md` to include this new entry, but that file wouldn't exist anymore since we deleted it, so the release would probably explode.

The other problem we have (recently noted in #2819, not directly although I think this is another case of that) is that a changelog like this bumps the root SDK module for basically no reason (no actual change) which by proxy bumps every service client module for equally no reason.

TL;DR not sure what to do here. We should honestly probably just not have a changelog for these types of deprecations for now.

**Review by lucix-aws — CHANGES_REQUESTED** [review]
Let's not merge this until we make a decision on ☝️

File: .changelog/4490529bdb8044068d99216cba424862.json (hunk 1 above)
Comment (wty-Bryant) [reply .changelog/4490529bdb8044068d99216cba424862.json:6]: involved modules are generated by changelog cmd according to which commit we use, here I passed the commit including both model and service api removal
